app/src/main/java/com/rockthevote/grommet/util/ValidationRegex.kt

```diff
@@ -0,0 +1,16 @@
+package com.rockthevote.grommet.util
+
+object ValidationRegex {
+    const val PHONE = "/[ [:punct:]]*\\d{3}[ [:punct:]]*\\d{3}[ [:punct:]]*\\d{4}\\D*/"
+    const val EMAIL = " /\\A[A-Z0-9_.&%+\\-‘]+@(?:[A-Z0-9\\-]+\\.)+(?:[A-Z]{2,25})\\z/ix"
```

Comment:
looks like we can probably ditch the extra email regex

![image](https://user-images.githubusercontent.com/1134161/87233545-814c6680-c396-11ea-9a1b-3d190f649776.png)

Reply:
seems like John, does want this implemented. I've left the ball in Alex's court to refine the regex, and John is aware.
![image](https://user-images.githubusercontent.com/1134161/87234175-7bf21a80-c39c-11ea-9b81-c1bd9032c5ef.png)

Reply:
new update, I guess Jon is fine w/ skipping email validation for now and dealing w/ it down the road if it's an issue

![image](https://user-images.githubusercontent.com/1134161/87234421-9dec9c80-c39e-11ea-9bff-94b46f5adc75.png)